README.md

```diff
@@ -1,34 +1,166 @@
+
+## Helpful Tools
+
+In addition to the formatting available in Markdown, our site includes some tools
+to help produce high-quality specifications.
+
+The source for these tools are in `_includes` and the styles are in `_sass/blocks`.
+
+### Image with Caption
+
+This renders a consistent, nicely-formatted image with a caption. Almost all
+images in a specification should use this layout.
+
+```
+{% include image.html src='path/to/image.png' caption='My Caption Here' %}
+```
+
+will render something like:
+
+![Image with caption](./readme-images/image-with-caption-example.png)
+
+
+### Desktop and Mobile Comparison
+
+Often, we will have images of the same content on Desktop and on Mobile. These can easily be displayed in a tabbed view with:
+
+```
+{% include mobile-and-desktop.html desktop='path/to/desktop.png' mobile='path/to/mobile.png' caption='My Caption Here' %}
+```
+
+### Coming Soon
+
+Blocks that are coming soon:
+
+* Color Palette links (so you can just say 'navy' and have a link to the definition of 'navy').
+* Font Stack links (so you can just say 'the Vitesse stack' and have a link to the stack, fallbacks, and example images).
+
+## Specification Content
+
+Good specifications include a mix of graphics and text to both explain and show the specifications.
+
+The specification should be complete enough that two different people with no foreknowledge of
+the design should be able to independently create visually-identical implementations
+of the specification without having to ask the original designer for clarifications.
+They should also be flexible enough that they can be adapted to a wide variety of use cases
+and design considerations.
+
+The specification should explain:
+
+1. Underlying Design principles
 2. The Why, The What, and The How
 3. Fonts
-* Font styles. This means font family, weight, size, font style, font stack (including fall backs)
-* Font styles should be concise. We encourage only a few per font styles and we highly encourage using the same font styles.
+    * Font styles. This means font family, weight, size, font style, font stack (including fall backs)
+    * Font styles should be concise. We encourage only a few per font styles and we highly encourage using the same font styles.
 4. Breakpoints
-* And differences at those breakpoints
+    * And differences at those breakpoints
 5. Colors
-* Different shades of each, when and where to each, and what colors of text are allowed with each.
-* Sizing & Measurements
-* Present measurements with consideration to the CSS box model.
-* When appropriate, the measurement should or can be a percentage. Please be aware of trying to make designs fluid.
-* Please be comfortable with using and understanding flexbox.
+    * Different shades of each, when and where to each, and what colors of text are allowed with each.
+    * Sizing & Measurements
```

Comment:
Why are lines 93-96 under the "Color" section when they look like they should be under some sort of "structure" section?

Reply:
I believe Sizing & Measurements is supposed to be a new section, like 6.

Reply:
@blakemorgan i think that's a mistake, we should go ahead and fix it, but we can do it after merging this branch i think

Reply:
I wasn't reading much, I was just trying to fix the formatting by indenting all of the bullets and leaving all of the numbered points at the top level. It's easy enough to fix.